l10n_it_fatturapa/models/ir_attachment.py

```diff
@@ -104,16 +104,24 @@ def get_xml_string(self):
             raise UserError(_("Invalid xml %s.") % e.args)
 
     def get_fattura_elettronica_preview(self):
-        xsl_path = get_resource_path(
-            "l10n_it_fatturapa",
-            "data",
-            self.env.company.fatturapa_preview_style,
-        )
-        xslt = ET.parse(xsl_path)
         xml_string = self.get_xml_string()
         xml_file = BytesIO(xml_string)
         recovering_parser = ET.XMLParser(recover=True)
         dom = ET.parse(xml_file, parser=recovering_parser)
+        # identify the schema to use by looking at the root element tag
+        root = dom.getroot()
+        index = root.tag.rfind("}") + 1
+        root_tag = root.tag[index:]
+
+        if root_tag == "FatturaElettronicaSemplificata":
+            preview_style = self.env.company.fatturapa_simple_preview_style
+        elif root_tag == "FatturaElettronica":
+            preview_style = self.env.company.fatturapa_preview_style
+        else:
+            raise ValidationError(_("Unexpected root element: {}".format(root_tag)))
```

Comment:
Non ho guardato il resto della PR, ho solo notato questo leggendo le email
```suggestion
            raise ValidationError(_("Unexpected root element: {}").format(root_tag))
```

Reply:
Su richiesta di @primes2h ho cambiato la stringa. I traduttori si aspettano %s oppure %(name)s.

Reply:
> Su richiesta di @primes2h ho cambiato la stringa. I traduttori si aspettano %s oppure %(name)s.

Grazie, ancora meglio sarebbe impostare la variabile come parametro.
È un metodo più "resiliente" in caso di errori. Se dovesse mancare il segnaposto nella traduzione, ripiega sulla stringa sorgente. 

Vedi:
https://www.odoo.com/documentation/14.0/developer/howtos/translations.html#variables

Reply:
Dici così?  Considera che il 99% delle stringhe attuali nei moduli l10n_italy sono fatte nell'altro modo...